src/StdChains.sol

```diff
@@ -171,6 +177,10 @@ abstract contract StdChains {
         return chain;
     }
 
+    function setUseDefaultRpcUrls(bool useDefault) internal {
```

Comment:
Similarly 
```suggestion
    function setFallbackToDefaultRpcUrls(bool useDefault) internal {
```
(this file has a lot going on so just trying to lean towards keeping things explicit 😅)

Reply:
changed